main.py

```diff
@@ -1,142 +1,39 @@
 """
 Created on 14/12/2022
 
-@author Giulia, Giuseppe
 
-"""
-
-import random
-
-from locales.i18n import choose_language, set_locale
-
-def carica_parole():
-    livello = input(_('Scegli un livello di gioco:\n1. facile\n2. medio\n3. difficile\n\n'))
-    lista_di_parole = []
-    
-    if livello == '1' :
-        with open('src/db_words/parole_facili.txt', 'r') as f:
-            for line in f:
-                parola = line.rstrip('\n')
-                lista_di_parole.append(parola)
-     
-    elif livello == '2' :
-        with open('src/db_words/parole_medie.txt', 'r') as f:
-            for line in f:
-                parola = line.rstrip('\n')
-                lista_di_parole.append(parola)
-
-    elif livello == '3' :
-        with open('src/db_words/parole_difficili.txt', 'r') as f:
-            for line in f:
-                parola = line.rstrip('\n')
-                lista_di_parole.append(parola)
-
-    return lista_di_parole
-     
+@author Giulia, Giuseppe, Tommaso
 
-
-def parola_random(lista_di_parole):
-    return random.choice(lista_di_parole)
-
-def visualizza_parola(lista_lettere):
-    print(' '.join(lettera for lettera in lista_lettere))
-
-def aggiorna_lettere_indovinate(parola, scelta, lettere_indovinate):
-    for i, lettera in enumerate(parola):
-        if lettera == scelta:
-            lettere_indovinate[i] = lettera
-    return lettere_indovinate
-
-def show_player_life():
-
-    life = """
----
 """
-    yield life
-
-    life = """
-|----
-|
-|
-|
-|
----
-"""
-    yield life
-    life = """
-|----0
-|    |
-|    |
-|    |
-|    |
----
-"""
-    yield life
-
-    life = """
-|----O
-|    |
-|   /|\
-|    |
-|    |
----
-"""
-
-    yield life
-
-    life = """
-|----O
-|    |
-|   /|\
-|    |
-|   /|\
----
-"""
-
-    yield life
-    life = """
-
-|--\
-|   \O
-|    |1
-|   /|\
-|    |
---- /|\
-"""
-    yield life
-
 
-# fai scegliere all'utente la lingua
-lingua = choose_language()
 
-# impostala come lingua per questa partita
-_ = set_locale(lingua)
+from src.classModules import *
```

Comment:
Anche se così è più conciso, ti consiglierei di importare in maniera esplicita tutte le classi che ti interessano